### PR TITLE
Support for django-sheerlike: less code this time

### PR DESCRIPTION
**Pull request by rosskarchner**
I spent time today figuring out how to support a true global request object in django, AND a non-asinine ways to allow relative template imports. django-sheerlike (pull request pending) now supports both of these.

TLDR;  the only necessary template changes left relate to adding 'else' clauses to inline 'if' statements.

This is needed to suppress warnings that you will only see in debug mode. It's worth fixing because, when you're *in* debug mode, the warnings sometime appear inside text fields, which can interfere with filtering, etc.

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Makes sense to me :+1:

File: _includes/post-macros.html [hunk 1]
```diff
@@ -298,7 +298,7 @@ <h1 class="summary_heading">{{ post.title|safe }}</h1>
 <div class="expandable
             expandable__filters
             expandable__padded
-            {{ 'expandable__expanded' if is_expanded }}
+            {{ 'expandable__expanded' if is_expanded else ''}}
```
Comment (anselmbradford) [inline _includes/post-macros.html]: Very minor, but the trailing space is missing in all the changes in this file (they are being added in, e.g. _includes/macros/util/format.html).

Comment (rosskarchner) [inline _includes/post-macros.html]: better?

Comment (anselmbradford) [inline _includes/post-macros.html]: :+1:

**Comment by anselmbradford** [comment]
Please rebase onto flapjack branch to move `./_includes/` paths to `./src/_includes/`. Otherwise, :+1:

**Pull request closed without merging**